prov/efa/src/rxr/rxr_rma.c

```diff
@@ -462,7 +463,12 @@ ssize_t rxr_rma_post_write(struct rxr_ep *ep, struct rxr_tx_entry *tx_entry)
 	if (OFI_UNLIKELY(err))
 		return err;
 
-	return rxr_pkt_post_ctrl_or_queue(ep, RXR_TX_ENTRY, tx_entry, RXR_LONG_RTW_PKT, 0);
+	ctrl_type = delivery_complete_requested ?
+		RXR_DC_LONG_RTW_PKT : RXR_LONG_RTW_PKT;
+	return rxr_pkt_post_ctrl_or_queue(ep,
+					  RXR_TX_ENTRY,
+					  tx_entry,
+					  ctrl_type, 0);
```

Comment:
You can just leave them in the same line.

Reply:
Comment addressed, resolving.